From 54367a0ec537b4bc361783b9b375b9cbd7d48483 Mon Sep 17 00:00:00 2001
From: Kenichi Handa <handa@m17n.org>
Date: Sun, 4 Mar 2012 21:05:23 +0900
Subject: quail/indian.el (indian-tlg-base-table): Fix typo; dev -> tlg.

In quail/indian.el, indian-dev-base-table has been corrected to
indian-tlg-base-table.

Applied-By: Rob Browning <rlb@defaultvalue.org>
Origin: upstream, commit: 141162dace9e05123a73dfe6d8be921840ff2416
---
 leim/ChangeLog       |    4 ++++
 leim/quail/indian.el |    2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/leim/ChangeLog b/leim/ChangeLog
index 1a8c37a..7ca7270 100644
--- a/leim/ChangeLog
+++ b/leim/ChangeLog
@@ -1,3 +1,7 @@
+2012-03-04  Kenichi Handa  <handa@m17n.org>
+
+	* quail/indian.el (indian-tlg-base-table): Fix typo; dev -> tlg.
+
 2012-01-19  Chong Yidong  <cyd@gnu.org>
 
 	* Version 23.4 released.
diff --git a/leim/quail/indian.el b/leim/quail/indian.el
index d92fef5..6d4ecbf 100644
--- a/leim/quail/indian.el
+++ b/leim/quail/indian.el
@@ -269,7 +269,7 @@
 (if nil
     (quail-define-package "telugu-inscript" "Telugu" "TlgIS" t "Telugu keyboard Inscript"))
 (quail-define-inscript-package
- indian-dev-base-table inscript-dev-keytable
+ indian-tlg-base-table inscript-dev-keytable
  "telugu-inscript" "Telugu" "TlgIS"
  "Telugu keyboard Inscript.")
 
